pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.fr-fr.md

```diff
@@ -40,10 +64,28 @@ $ kubectl get nodes -o jsonpath='{ $.items[*].status.addresses[?(@.type=="Intern
 51.77.204.175 51.77.205.79
 ```
 
+#### Your cluster is installed with Private Network with a OVHcloud Internet Gateway (or a defaultVrackGateway if you are using OVHcloud APIv6)
+
+Because your nodes are configured to be routed by the private network gateway, you need to add the gateway IP address to the ACLs.
+By using Public Cloud Gateway through our Managed Kubernetes Service, Public IPs on nodes are only for management purposes: [MKS Known Limits](/pages/public_cloud/containers_orchestration/managed_kubernetes/known-limits)
+
+You can get your OVHcloud Internet Gateway's Public IP by navigating through the OVHcloud Manager:
+
+Public Cloud > Select your tenant > Network / Gateway > Public IP
+
+You can also get your OVHcloud Internet Gateway's Public IP by using our APIs:
+
+> [!api]
+>
+> @api {v1} /cloud GET  /cloud/project/{serviceName}/region/{regionName}/gateway/{id}
+>
+
+You can find more details about how using OVHcloud APIs with this guide: [First Steps with the OVHcloud APIs](/pages/manage_and_operate/api/first-steps)
+
 Click on the *Manage Access* menu of our newly created partition:
 ![Manage Access of the NFS partition](images/manage-nfs-partition-access.png){.thumbnail}
 
-Add our nodes IP one by one:
+Add either the nodes' IPs one by one or the Gateway's Public IP depending on your configuration:
```

Comment:
Why we need to add nodes's IPs here?

Reply:
If the cluster is not using Private Network + Gateway combo, you still need to add nodes' Public IPs to the HA-NAS ACL.
Maybe I missed smthg ?

Reply:
OK, I'm the one who's not understood.
Solve!